### DISCUSSION TITLE
Crossing lines in draw.io

### DISCUSSION THREAD

**Discussion by Braincramps** [jgraph/drawio-desktop#1416] [Q&A]
New to this program and trying to get the lines that cross over to be easier to differentiate.  In cad/viso and other programs one of the lines forms a small bridge/hump as it crosses the other line, so you know which is which.  Is this an option in this program?  I can't seem to find it.  Thanks in advance for your help.
Mike

**Comment by tmal-1** [comment]
These are referred to as Line Jumps within draw.io

There appears to be a bug which hides this option by default.

If I select a line the Style dialogue appears as in my left screenshot. I have highlighted the line type with a red box.

If I click on that line type and select straight line (the type it already had) the Style dialogue changes as in my right screenshot. If have highlighted an additional row of controls with a red box.

The menu that currently says 'None' allows you to choose the type of Line Jump. The value that currently says '6 pt' allows you to vary the size of the Line Jump.

<img width="246" alt="Screenshot 2023-08-23 at 16 18 09" src="https://github.com/jgraph/drawio-desktop/assets/122500060/b2633dda-fec2-42c3-9cf2-b7b88af8bc07">
<img width="242" alt="Screenshot 2023-08-23 at 16 18 34" src="https://github.com/jgraph/drawio-desktop/assets/122500060/bafe31fd-fac0-49d8-a581-1f5ed08d32b3">
